ARCHITECTURE.md

```diff
@@ -0,0 +1,41 @@
+# Architecture of devSwag.io
+
+## Overview
+
+[devSwag.io](https://devswag.io/) uses [`gulp.js`](https://gulpjs.com/) to build the website. The website is composed of [`pug.js`](https://github.com/pugjs/pug) to template the front end of the website, [`Styl`](https://github.com/tj/styl) to add styling, and the backend is a simple JSON file called `data.json` that stores each swag item in a JSON format.
```

Comment:
- It's `Gulp`, not `gulp.js`
- It's `Pug`, not `pug.js`
- We use Stylus (https://github.com/stylus/stylus) to template style sheets.